functions/consumer/rsocket-consumer/src/main/java/org/springframework/cloud/fn/consumer/rsocket/RsocketConsumerConfiguration.java

```diff
@@ -34,17 +34,15 @@ public class RsocketConsumerConfiguration {
 	@Bean
 	public Function<Flux<Message<?>>, Mono<Void>> rsocketConsumer(RSocketRequester.Builder builder,
 																RsocketConsumerProperties rsocketConsumerProperties) {
-		final Mono<RSocketRequester> rSocketRequester =
-				rsocketConsumerProperties.getUri() != null ? builder.connectWebSocket(rsocketConsumerProperties.getUri()).cache() :
```

Comment:
Ahhh yeh, this is a nice simplification of the RSocket API - they dropped the Mono. 👍🏻

Reply:
Well, it still there, but handled internally.

Reply:
Yep, it has to still be there but now the API  avoid the cookie cutter code that everyone did for Mono.cache->flatMap